engine/phone/scripts/jack_gossip.asm

```diff
@@ -0,0 +1,68 @@
+JackTriviaScript:
+	random 11
+	ifequalfwd  0, .Thunder
+	ifequalfwd  1, .Rollout
+	ifequalfwd  2, .Solarbeam
+	ifequalfwd  3, .Stomp
+	ifequalfwd  4, .Gust
+	ifequalfwd  5, .Twister
+	ifequalfwd  6, .Earthquake
+	ifequalfwd  7, .Magnitude
+	ifequalfwd  8, .Sandstorm
+	ifequalfwd  9, .SunnyDay
+	ifequalfwd 10, .RainDance
+
+.Thunder:
+	farwritetext JackThunderTriviaText
+	promptbutton
+	sjump PhoneScript_HangupText_Male
+
+.Rollout:
+	farwritetext JackRolloutTriviaText
+	promptbutton
+	sjump PhoneScript_HangupText_Male
+
+.Solarbeam:
+	farwritetext JackSolarbeamTriviaText
+	promptbutton
+	sjump PhoneScript_HangupText_Male
+
+.Stomp:
+	farwritetext JackStompTriviaText
+	promptbutton
+	sjump PhoneScript_HangupText_Male
+
+.Gust:
+	farwritetext JackGustTriviaText
+	promptbutton
+	sjump PhoneScript_HangupText_Male
+
+.Twister:
+	farwritetext JackTwisterTriviaText
+	promptbutton
+	sjump PhoneScript_HangupText_Male
```

Comment:
These labels should be relabeled since Twister is not a move in 9-bit. Need recommendations on what you would like. You may also want to revisit the actual text as well.